modules/Behavior.js

```diff
@@ -23,7 +23,7 @@
     'render:begin': 'prerender',
     'render:complete': 'postrender',
     'initialize:begin':  '_preinitialize',
-    'initialize:complete': '_postinitialize'
+    'initialize:complete': '__postinitialize'
```

Comment:
Add a special listener to initialize:complete that does what __postinitialize does.